modules/virt-about-aaq-operator.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-understanding-aaq-operator.adoc              
+
+:_mod-docs-content-type: CONCEPT                                    
+[id="virt-about-aaq-operator_{context}"]                                   
+= About the AAQ Operator
+
+
+The AAQ Operator provides more flexible and extensible quota management compared to the native `ResourceQuota` object in the {product-title} platform.
+
+In a multi-tenant cluster environment, where multiple workloads operate on shared infrastructure and resources, using the Kubernetes native `ResourceQuota` object to limit aggregate CPU and memory consumption presents infrastructure overhead and live migration challenges for {VirtProductName} workloads. 
+
+{VirtProductName} requires significant compute resource allocation to handle virtual machine (VM) live migrations and manage VM infrastructure overhead. When upgrading {VirtProductName}, it is necessary to migrate VMs to upgrade the `virt-launcher` pod. However, migrating a VM in the presence of a resource quota can cause the migration to fail and subsequently cause the upgrade to fail. 
+
+With AAQ, you can allocate resources for VMs without interfering with cluster-level activities such as upgrades and node maintenance. The AAQ Operator also supports non-compute resources. This eliminates the need to manage both the native resource quota and AAQ API objects separately.
+
+
+.AAQ Operator controller and custom resources
+The AAQ Operator introduces two new API objects defined as custom resource definitions (CRDs) for managing alternative quota implementations across multiple namespaces:
+
+* `ApplicationAwareResourceQuota`: Sets aggregate quota restrictions enforced per namespace. The `ApplicationAwareResourceQuota` API is compatible with the native `ResourceQuota` object and shares the same specification and status definitions.
++
+.Example manifest
+[source,yaml]
+----
+apiVersion: aaq.kubevirt.io/v1alpha1
+kind: ApplicationAwareResourceQuota
+metadata:
+  name: example-resource-quota
+spec:
+  hard:
+    requests.memory: 1Gi 
+    limits.memory: 1Gi
+    requests.cpu/vmi: "1" # <1>
+    requests.memory/vmi: 1Gi # <2>
+# ...
+----
+<1> The maximum amount of CPU that is allowed for VM workloads in the default namespace.
+<2> The maximum amount of RAM that is allowed for VM workloads in the default namespace.
+
+* `ApplicationAwareClusterResourceQuota`: Mirrors the `ApplicationAwareResourceQuota` object at a cluster scope. It is compatible with the native `ClusterResourceQuota` API object and shares the same specification and status definitions. AAQ cluster quotas are not namespaced. When creating an AAQ cluster quota, you can select multiple namespaces based on annotation selection, label selection, or both by editing the `spec.selector.labels` or `spec.selector.annotations` fields.
```

Comment:
I find this sentence "AAQ cluster quotas are not namespace." confusing. Perhaps it's not needed? We already said that this is a cluster object.